Changelog.md

```diff
@@ -3,6 +3,7 @@
 - Adds `response` computed property to `Error` type, which yields a Response object if available.
 - Added URLEncodedInURL to ParameterEncoding.
 - Adds convenience `endpointByAdding` method.
+- **Breaking Change** Remove our own implementation of `ParameterEncoding` and make it a `public typealias` of `Alamofire.ParameterEncoding`.
```

Comment:
I was suggested by the removed `toAlamofire()` function which our `ParameterEncoding` had. 🤔But it was internal so I'm fine with removing the breaking change alias ✌️